azure-pipelines.yml

```diff
@@ -30,7 +30,7 @@ stages:
     displayName: "kvmtest-t0-part1"
     timeoutInMinutes: 400
     condition: and(succeeded(), eq(variables.RUN_CLASSICAL_TEST, 'YES'))
-    continueOnError: true
+    continueOnError: false
```

Comment:
If continueOnError set to false and t0_part1 failed, will job kvmtest-t0 run? which is depended on t0_part1? will it be waiting for timeout? I can't recall why set continueOnError to True, wonder if there is particular reason to do it so.

Reply:
If continueOnError set to false and t0_part1 failed, job kvmtest-t0 will run.
kvmtest-t0 depends on t0_part1, it won't wait and won't timeout, it's triggered immediately t0_part1 and to_part2 finished.
continueOnError: true only affects the result when the job is failed, "failed" -> "partiallySuccess".